ipatests/test_integration/test_replica_promotion.py

```diff
@@ -47,6 +48,32 @@ def test_kra_install_master(self):
         assert(found > 0), result2.stdout_text
 
 
+def test_sssd_config_allows_ipaapi_access_to_ifp(host):
+    '''Tests that the sssd configuration allows the ipaapi user to access
+    ifp
+
+    :param host the machine on which to test that sssd allows ipaapi
+    access to ifp
+    '''
+    contents = host.get_file_contents(paths.SSSD_CONF, encoding='utf-8')
+    # Parse the sssd config as a config file
+    config = ConfigParser()
```

Comment:
Strictly speaking, yes, the config is an INI-style configuration file. There is also an SSSD python API and that may be a better long-term solution. Maybe something approaching:

```
        sssdconfig = SSSDConfig.SSSDConfig()
        sssdconfig.import_config()

        sssdconfig.get_service('ifp')

        uids = ifp.get_option('allowed_uids').split(',')
        assert 'ipaapi' in uids
```

Reply:
I believe this is addressed in commit adc5e44.